tests/sentry/db/models/test_utils.py

```diff
@@ -6,26 +6,37 @@
 class SlugifyInstanceTest(TestCase):
     def test_no_conflict(self):
         org = Organization(name="matt")
-        slugify_instance(org, "matt")
+        slugify_instance(org, org.name)
         assert org.slug == "matt"
-        assert not Organization.objects.filter(slug="matt").exists()
```

Comment:
Lol! I'm confused. How did this assert pass before?

Reply:
I was confused at first too, basically b/c you're not doing `Model.objects.create(...)` and instead doing `Model()` directly it doesn't write to the DB.

Reply:
It sounds very stupid. Lol! But do you want to keep one of them? Maybe there was a reason the author wanted to test it that we missed? 😛

Reply:
They did b/c otherwise the `save` method would run and call `slugify_instance`, so the org couldn't have a bad slug